tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "module": "CommonJS",
+    "target": "ES6",
+    "sourceMap": true
+  },
+  "exclude": [
+    "node_modules"
```

Comment:
Can you explain why this is needed? :)

Reply:
I do not know why source maps are needed other than the general statement that they make debugging easier so I left them as part of the default template.
The same goes for the exclusion of `node_modules` because I assumed that it is generally not a got idea to let typescript (re)compile included files. I am aware of the default exclusion when not explicitly specified but defaults can change and this is the only needed exclusion atm.